smart-contracts/scripts/deployments/deploy-from-package.js

```diff
@@ -0,0 +1,58 @@
+const { ethers, upgrades, unlock } = require('hardhat')
+const { networks } = require('@unlock-protocol/networks')
+const { getNetworkName } = require('../../helpers/network')
+const { getImplementationAddress } = require('@openzeppelin/upgrades-core')
+
+async function main(owner) {
+  const [deployer] = await ethers.getSigners()
+  console.log(`Deploying from ${deployer.address}.`)
+
+  // fetch chain info
+  const chainId = await deployer.getChainId()
+  const networkName = getNetworkName(chainId)
+
+  await unlock.deployProtocol();
```

Comment:
that task was intended mostly for local dev, not sure we should use it for prod purposes